highcharts/src/test/scala/react/highcharts/ChartTest.scala

```diff
@@ -22,9 +22,11 @@ object MainTest extends TestSuite {
       ReactTestUtils.withRenderedIntoDocument(chart.toUnmounted) { m =>
         val html           = m.outerHtmlScrubbed()
         val normalizedHTML =
-          html.replaceAll(" id=\"highcharts-\\w{7}-\\d-?\"", "")
+          html
+            .replaceAll(" id=\"highcharts-\\w{7}-\\d-?\"", "")
+            .replaceAll("Created with Highcharts \\d+\\.\\d+\\.\\d+", "Created with Highcharts x.y.z")
```

Comment:
Seems that it was including the version number, so every update would break the test 🙃

Reply:
That was "copy+paste testing". Thank you for fixing!